src/workerManager.ts

```diff
@@ -40,7 +46,9 @@ export class WorkerManager {
 	}
 
 	dispose(): void {
-		clearInterval(this._idleCheckInterval);
+		if (this._idleCheckInterval) {

```

Comment:
fyi - those checks before running `clearTimeout|Interval` are not necessary - https://html.spec.whatwg.org/multipage/webappapis.html#dom-cleartimeout